pkgs/applications/office/pympress/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, python3Packages, pkgs, vlc }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pympress";
+  version = "1.2.0";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "10v1857kbd57s3iq5b9hb5k2ivhx5jcmjzjc0d3lw9mvl280yg86";
+  };
+
+  /* with check, segmentation fault:
+  _gtk_style_provider_private_get_settings: assertion 'GTK_IS_STYLE_PROVIDER_PRIVATE (provider)' failed
+  */
+  doCheck = false;
+
+  nativeBuildInputs = with pkgs; [
+    wrapGAppsHook
+  ];
+
+  buildInputs = with pkgs; [
+    gtk3
+    gobject-introspection
+    libcanberra-gtk3
+    poppler_gi
+    vlc
+  ];
+
+  propagatedBuildInputs =  with python3Packages; [
+    pycairo
+    pygobject3
+    python-vlc
+  ];
+
+  postInstall = ''
```

Comment:
This can now be removed.

Reply:
Empirically, this is still needed :/

Reply:
If this is still needed then something is still broken.

Reply:
fixed!